### PR TITLE
Create new task

### PR DESCRIPTION
**Pull request by TueAnhAnh**
ready

### CODE REVIEW

**Comment by TueAnhAnh** [comment]
ready

File: app/Http/Controllers/TaskController.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers;
 
+use App\Http\Requests\StoreRequest;
```
Comment (quoctd-0607) [inline app/Http/Controllers/TaskController.php:5]: TaskRequest thay vì StoreRequest nhé.

**Comment by TueAnhAnh** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**